client/src/components/App.jsx

```diff
@@ -68,16 +71,17 @@ class App extends React.Component {
                 <li>
                   <Link to="/trade">Trade</Link>
                 </li>
-                <li>
-                  <Link to="/leaderboard">Leaderboard</Link>
-                </li>
               </ul>
             </nav>
           </div>
           <Switch>
+            <Route exact path="/" component={Leaderboard} />
             <Route exact path="/leaderboard" component={Leaderboard} />
+            <Route exact path="/portfolio" component={Portfolio} />
+            <Route exact path="/stock-search" component={StockSearch} />
```

Comment:
In my recent pull request I had refactored this to include the search page, but we can just merge this and I will go in and replace the stocksearch component.